src/database.ts

```diff
@@ -821,7 +822,14 @@ class Database extends common.GrpcServiceObject {
         callback!(err as ServiceError, null, undefined);
         return;
       }
+
+      const span = getActiveOrNoopSpan();
+      span.addEvent('Creating BatchTransaction');
+
       const transaction = this.batchTransaction({session: session!}, options);
+      span.addEvent('BatchTransaction Creation Done', {
```

Comment:
This Span needs to move to the "transaction.begin" called below. Till the time transaction.begin is called, there will not be any transaction "id"

Reply:
Good point, thank you!